### PR TITLE
LIT 6193 6576 6616 617 6618 6619

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Some updates and a few new records around Dersana Madhane Alam, which is very complex and needs more study. I created a record (LIT6198) for versions that do not coincide with the one edited (more or less, it is LIT6193 - i change the title to dissimilate it from another collecrion). LIT6616 and LIT6617 are two pieces that circulate through two collections at least. LIT6617 is for a series of short homilies sometimes appended to/infixed into LIT6618 works. If it is ok, I will make a couple of revisions in a few relevant mss (BNFet116 and one or two more).

### CODE REVIEW

File: new/LIT6616HomilyPassionIntro.xml [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>The text is usually preceded by <ref type="work" corresp="LIT6616HomilyPassionIntro"/>.</p>
```
Comment (thea-m) [inline new/LIT6616HomilyPassionIntro.xml:33]: This sentence doesn't make a lot of sense in this record.

File: new/LIT6618DersanaMadhane.xml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>Various collections related to <ref type="work" corresp="LIT6193HomiliaryPassions"/>, relevant for the
```
Comment (thea-m) [inline new/LIT6618DersanaMadhane.xml:35]: So is this a general record?

File: new/LIT6619HomiliesFeastSaviour.xml [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
```
Comment (thea-m) [inline new/LIT6619HomiliesFeastSaviour.xml:33]: Could you include an abstract?

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you! I think it will be very helpful. Please go ahead with the manuscripts.

File: new/LIT6616HomilyPassionIntro.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply new/LIT6616HomilyPassionIntro.xml:33]: I would leave it, it gives a glimpse into its circulation mode.

File: new/LIT6618DersanaMadhane.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply new/LIT6618DersanaMadhane.xml:35]: yes.

File: new/LIT6619HomiliesFeastSaviour.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply new/LIT6619HomiliesFeastSaviour.xml:33]: done.

File: new/LIT6616HomilyPassionIntro.xml (hunk 1 above)
Comment (eu-genia) [reply new/LIT6616HomilyPassionIntro.xml:33]: You did not understand: the text is self-referring. It says that LIT6616HomilyPassionIntro is usually preceded by LIT6616HomilyPassionIntro, this cannot be what you wanted to say

Comment (eu-genia) [reply new/LIT6616HomilyPassionIntro.xml:33]: I think you mean LIT6617HomilyPassionIntro

Comment (DenisNosnitsin1970) [reply new/LIT6616HomilyPassionIntro.xml]: corrected.

**Review by eu-genia — APPROVED** [review]

**Review by thea-m — APPROVED** [review]

**Comment by eu-genia** [comment]
https://github.com/BetaMasaheft/Documentation/issues/1500
https://github.com/BetaMasaheft/Documentation/issues/2077

**Comment by eu-genia** [comment]
@DenisNosnitsin1970 on going through these issues: you say you created LIT6198 ("for versions that do not coincide with the one edited") but it is nowhere to be found, can you please check and eventually commit the record?

**Comment by DenisNosnitsin1970** [comment]
I think all records considered necessary for that moment are in the commit " LIT 6193 6576 6616 617 6618 6619 #794". They all belong to "the cycle of the Saviour of the World" (the texts deriving from a Latin source is a different matter).  It may deviate from the initial plan, but I suggest to leave it unless there are obvious heavy mistakes. This is a very complex issue, with "telescopic" compositions that are parts of each other plus hanging together with "the cycle of Maba's Seyon", which is difficult to dig to the bottom.  If there will be more mss, we will see how it works. If someone wants to check it profoundly, simply have to take more or all witnesses across various collections, which now probably is not so necessary.

**Comment by eu-genia** [comment]
So when you wrote here above in the very first message "I created a record (LIT6198) for versions that do not coincide with the one edited "  which one were you referring to?

> Some updates and a few new records around Dersana Madhane Alam, which is very complex and needs more study. I created a record (LIT6198) for versions that do not coincide with the one edited (more or less, it is LIT6193 - i change the title to dissimilate it from another collecrion). LIT6616 and LIT6617 are two pieces that circulate through two collections at least. LIT6617 is for a series of short homilies sometimes appended to/infixed into LIT6618 works. If it is ok, I will make a couple of revisions in a few relevant mss (BNFet116 and one or two more).

**Comment by DenisNosnitsin1970** [comment]
Now, difficult to say. I repeat, all necessary records are in this commit.

**Comment by eu-genia** [comment]
I now discovered also LIT1291Dersan, it seems that the abstract there needs to be fixed as it refers to itself but probably should refer to LIT6193HomiliaryPassions https://github.com/BetaMasaheft/Works/pull/830

**Pull request merged**